contrib/vscode-debugging.md

```diff
@@ -0,0 +1,74 @@
+# VSCode Debugging
+
+## Setup
+
+First make sure your environment is setup by following the [building instructions](./building.md)
+
+Then you will need the following plugins for VSCode:
+* [C/C++ by Microsoft](https://github.com/Microsoft/vscode-cpptools)
+* [CodeLLDB](https://github.com/vadimcn/vscode-lldb)
+
+## Building
+
+In order to debug C++, you must first build the project in debug mode.  This can be done with the command
+
+```bash
+$ npm run rebuild --debug
```

Comment:
This won't work on v11+ (`develop`), will it?  We should state that here or provide alternative instructions.

Reply:
I did test this in develop actually.